Quadriga/src/main/java/edu/asu/spring/quadriga/qstore/impl/QStoreConnector.java

```diff
@@ -278,6 +280,29 @@ public String store(String xml) throws QStoreStorageException {
         return new AsyncResult<String>(executeNeo4jQuery("allNetworks", null, RELATION_EVENT));
     }
     
+    @Override
+    public String findStatementsWithConcepts(List<String> conceptUris) throws AsyncExecutionException {
+        Map<String, String> parameters = new HashMap<String, String>();
+       
+        if(conceptUris.size() == 0){
+            return null;
+        }
```

Comment:
move before the declaration of 'parameters'. No reason to instantiate it if conceptUris is empty.